src/imbi_api/endpoints/organizations.py

```
@@ -214,6 +214,48 @@ async def update_organization(
     return typing.cast(dict[str, typing.Any], org.model_dump())
 
 
+@organizations_router.get('/{slug}/members')
+async def list_organization_members(
+    slug: str,
+    auth: typing.Annotated[
```

Ruff (0.14.14)

[warning] Unused function argument: `auth`

(ARG001)